06_puppy/mohitnag/types.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import "sync"
+
+// Puppy defines the data structure corresponding to a Puppy
+type Puppy struct {
+	ID     uint32
+	Breed  string
+	Colour string
+	Value  string
+}
+
+// MemStore stores Puppy details with Puppy Id as Key and Puppy  as value
+type MemStore map[uint32]Puppy
```

Comment:
This should be called `MapStore`

Reply:
This will need to be updated in Lab 07 as well.

Reply:
although naming is subjective. Still i will rename it as MapStore to make it more consistent with SyncStore